看起來新版的 jwt 只支援 go 1.15 以上

# github.com/golang-jwt/jwt/v4
Error: ../../../go/pkg/mod/github.com/golang-jwt/jwt/v4@v4.0.0/ecdsa.go:135:4: r.FillBytes undefined (type *big.Int has no field or method FillBytes)
Error: ../../../go/pkg/mod/github.com/golang-jwt/jwt/v4@v4.0.0/ecdsa.go:136:4: s.FillBytes undefined (type *big.Int has no field or method FillBytes)
note: module requires Go 1.15

--- a/.github/workflows/go.yaml
+++ b/.github/workflows/go.yaml
@@ -34,7 +34,7 @@ jobs:
     strategy:
       matrix:
         platform: [ubuntu-20.04, macos-latest, windows-latest]
-        go-version: [1.14.x, 1.16.x]
+        go-version: [1.16.x]
     runs-on: ${{ matrix.platform }}
     name: integration tests
     env:

--- a/.github/workflows/go.yaml
+++ b/.github/workflows/go.yaml
@@ -9,7 +9,7 @@ jobs:
     strategy:
       matrix:
         platform: [ubuntu-20.04]
-        go-version: [1.14.x, 1.16.x]
+        go-version: [1.16.x]
     runs-on: ${{ matrix.platform }}
     name: Linters (Static Analysis) for Go
     env:

--- a/.github/workflows/go.yaml
+++ b/.github/workflows/go.yaml
@@ -9,7 +9,7 @@ jobs:
     strategy:
       matrix:
         platform: [ubuntu-20.04]
-        go-version: [1.16.x]
+        go-version: [1.16.x, 1.15.x]
     runs-on: ${{ matrix.platform }}
     name: Linters (Static Analysis) for Go
     env:
@@ -34,7 +34,7 @@ jobs:
     strategy:
       matrix:
         platform: [ubuntu-20.04, macos-latest, windows-latest]
-        go-version: [1.16.x]
+        go-version: [1.16.x, 1.15.x]
     runs-on: ${{ matrix.platform }}
     name: integration tests
     env:

